pallets/ctype/src/tests.rs

```diff
@@ -26,36 +26,36 @@ use crate::{self as ctype, mock::*};
 fn check_successful_ctype_creation() {
 	let creator = ALICE;
 
-	let operation = generate_base_ctype_creation_details();
+	let ctype_hash = get_ctype_hash::<Test>(true);
 
 	let builder = ExtBuilder::default();
 
 	let mut ext = builder.build(None);
 
 	// Write CType on chain
 	ext.execute_with(|| {
```

Comment:
We're not changing also these tests to use the changed syntax for externalitie?

Reply:
Didn't touch this wont change it!